server/models/Profile.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require("mongoose");
```

Comment:
I have minor accident before where i rename this branch resulting my first PR become closed automatically, therefore since i create this branch based on my first PR thus this model got included. This model doesn't need to be reviewed since i already made differente PR for this model here: https://github.com/rizperdana/fun-kickoff-project/pull/3

TL:DR This model accidentally included, please refrain from review it